### DISCUSSION TITLE
mbtiles are not displayed in Qfield iOS 2.8.3

### DISCUSSION THREAD

**Discussion by jcmoral90** [opengisch/QField#4677] [Bugs]
I have tried both from Qfield cloud and locally, and in both options the mbtiles file is not displayed. I have created the file in QGIS with jpg, 300 dpi, 12-18 zoom and quality 85. Any suggestions?

**Comment by nirvn** [comment]
@jcmoral90 , this will be fixed in QField 3.0, released in 96 hours. Hold tight :)
